bigquery-storage/api/BigQueryStorage.Samples/AppendRowsPending.cs

```diff
@@ -0,0 +1,141 @@
+        // Initialize a write stream for the specified table.
+        // When creating the stream, choose the type. Use the PENDING type to wait
+        //until the stream is committed before it is visible. See:
+        // https://cloud.google.com/bigquery/docs/reference/storage/rpc/google.cloud.bigquery.storage.v1#google.cloud.bigquery.storage.v1.WriteStream.Type
+        WriteStream stream = new WriteStream { Type = WriteStream.Types.Type.Pending };
+        TableName table = TableName.FromProjectDatasetTable(projectId, datasetName, tableName);
+        
+        stream = await bigQueryWriteClient.CreateWriteStreamAsync(table, stream);
+
+        // Initialize streaming call, retrieving the stream object
+        BigQueryWriteClient.AppendRowsStream rowAppender = bigQueryWriteClient.AppendRows();
+
+        // Sending requests and retrieving responses can be arbitrarily interleaved
+        // Exact sequence will depend on client/server behavior
+        // Create task to do something with responses from server
+        Task appendResultsHandlerTask = Task.Run(async () =>
+        {
+            AsyncResponseStream<AppendRowsResponse> appendRowResults = rowAppender.GetResponseStream();
+            while (await appendRowResults.MoveNextAsync())
+            {
+                AppendRowsResponse responseItem = appendRowResults.Current;
+                // Do something with responses
+                Console.WriteLine($"Response Append Result is {responseItem.AppendResult}");
+            }
+            // The response stream has completed
+        });
+
+        // Populate the data
+        // Create a batch of row data by appending proto2 serialized bytes to the
+        // SerializedRows repeated field.
+        ProtoData protoData = new ProtoData
+        {
+            // This is a TEMPORARY WORKAROUND via extension method using reflection till the feature request
+            // and PR for "ToProto()" in Protobuf library gets released.
+            WriterSchema = new ProtoSchema { ProtoDescriptor = CustomerRecord.Descriptor.ToProto() },
+            Rows = new ProtoRows
+            {
+                SerializedRows =
+                {
+                    new CustomerRecord{ CustomerNumber = 1, CustomerName = "Alice"}.ToByteString(),
+                    new CustomerRecord{ CustomerNumber = 2, CustomerName = "Bob"}.ToByteString()
+                }
+            }
+        };
+
+        // Initialize the append row request
+        // Set an offset to allow resuming this stream if the connection breaks.
+        // Keep track of which requests the server has acknowledged and resume the
+        // stream at the first non-acknowledged message. If the server has already
+        // processed a message with that offset, it will return an ALREADY_EXISTS
+        // error, which can be safely ignored.
+        // The first request must always have an offset of 0.
+        AppendRowsRequest appendRowrequest = new AppendRowsRequest
```

Comment:
Nit: `appendRowRequest` (the r of request should be capitalized).

Reply:
Done.